README.md

```diff
@@ -134,3 +142,5 @@ console.assert(foo.toString.includes("...")); // oops
 This basically makes the censorship toothless, so that you do not gain the encapsulation or memory usage benefits.
 
 You could try to patch around it by saying that only setting it to `true` works, and setting it to `false` or deleting the property does nothing. But that's just strange. We already have a mechanism for changing the state of an object in a non-reversible way: call a method on it. Thus, the `foo.censor()` proposal above (which has its own problems).
+
+Additionally, we want the author of the function to be the one in control of its censorship, not anyone with a reference to that function. This was discussed with respect to an API for `Error.prototype.stack` censorship in the January 2019 TC39 meeting.
```

Comment:
I think this works better in the "rejected alternatives" section.

Reply:
Agreed.